File: DerivcoUserModel/DerivcoUserModel.Data/Models/Address.cs
using Newtonsoft.Json;
using System;
using System.ComponentModel.DataAnnotations;
using System.Linq;

namespace DerivcoUserModel.Data.Models
{
    public class Address
    {
        [Key]
        public int Id { get; set; }
        public string LineOne { get; set; }
        public string LineTwo { get; set; }
        public string City { get; set; }
        public string Country { get; set; }
        public int? PostalCode { get; set; }
        public DateTime CreatedAt { get; set; }
        public DateTime UpdatedAt { get; set; }
    }
}

File: DerivcoUserModel/DerivcoUserModel.App/ClientApp/models/Address.ts
export class Address {
    constructor() {
        this.id = 0
        this.lineOne = ""
        this.lineTwo = ""
        this.city = ""
        this.country = ""
        this.postalCode = ""
        this.createdAt = new Date()
        this.updatedAt = new Date()
    }

    id: number
    lineOne: string
    lineTwo: string
    city: string
    country: string
    postalCode: string
    createdAt: Date
    updatedAt: Date
}

export default Address